src/OrchardCore.Modules/OrchardCore.Workflows/package.json

```diff
@@ -5,6 +5,6 @@
     "bootstrap": "4.5.2",
     "@types/bootstrap": "4.5.0",
     "jsplumb": "2.14.6",
-    "@types/jquery": "3.5.1"
+    "@types/jquery": "3.5.2"
```

Comment:
Confused. I thought we were still on `jQuery` 3.5.1 ?

Is the types upgrade fixing some issues with the jquery 3.5.1 type definitions?

Reply:
The version of `@types/jquery` is not always tied to the version of jquery.